### PR TITLE
fix typos.

### PR DESCRIPTION
**Pull request by haiker2011**
Signed-off-by: sunhaizhou [孙海洲] <sunhaizhou@kingsoft.cn>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
fix typos `for`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/convert.go [hunk 1]
```diff
@@ -261,7 +261,7 @@ func EnumSlice(val, sep string, enumValMap map[string]int32) ([]int32, error) {
 }
 
 /*
-	Support fot google.protobuf.wrappers on top of primitive types
```
Comment (RainbowMango) [inline vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/convert.go:264]: Maybe we should not manually update things from `vendor`. 
You can file a PR directly on `github.com/grpc-ecosystem/grpc-gateway`.

**Comment by RainbowMango** [comment]
Hi @haiker2011 Thanks for doing this. I'll trigger the test, If I remember right, the test would fail since the vendor is not allowed to update. Otherwise, LGTM.

File: vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/convert.go (hunk 1 above)
Comment (haiker2011) [reply vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/convert.go:264]: ok

**Comment by RainbowMango** [comment]
So @haiker2011 , could you please make changes only on the `hack/README.md`?
You can do that by following commands on branch `fix-typos`:
```
git reset HEAD~1
git add hack/README.md
git commit -s -m "fix typo"
git checkout -f 
git push origin fix-typos:fix-typos -f
```

**Comment by haiker2011** [comment]
@RainbowMango I have change

**Review by lfbear — APPROVED** [review]

**Comment by RainbowMango** [comment]
/hold 
for I see you just pushed an `anonymous` commit. Can you help squash the two commits to one?

**Comment by RainbowMango** [comment]
I know, you sined the commits with `sunhaizhou@kingsoft.cn` , but I'm afraid this email didn't set [in your profile](https://github.com/settings/emails).

**Comment by RainbowMango** [comment]
```
commit cfa86610d6f826a9cfaa0c03ac5e147cb8a9888d
Author: sunhaizhou [孙海洲] <sunhaizhou@kingsoft.cn>
Date:   Wed Dec 22 14:30:02 2021 +0800

    unfix vendor typos.
    
    Signed-off-by: sunhaizhou [孙海洲] <sunhaizhou@kingsoft.cn>

commit ca127818df28e7744cf1e3368fde3cab03b8c5ef
Author: sunhaizhou [孙海洲] <sunhaizhou@kingsoft.cn>
Date:   Wed Dec 22 13:11:10 2021 +0800

    fix typos.
    
    Signed-off-by: sunhaizhou [孙海洲] <sunhaizhou@kingsoft.cn>
```
By the way, we usually don't use Chinese names in commits.

**Comment by haiker2011** [comment]
@RainbowMango I fix my information , then make new PR.

**Pull request closed without merging**